src/common/dateHelpers.js

```diff
@@ -13,38 +12,133 @@ import { translate } from '../components/i18n/i18n';
  *
  * @returns {string|Date}
  */
-const getCurrentDate = () =>
-  (helpers.TEST_MODE && moment.utc('20190720').toDate()) ||
-  (helpers.DEV_MODE &&
-    process.env.REACT_APP_DEBUG_DEFAULT_DATETIME &&
-    moment.utc(process.env.REACT_APP_DEBUG_DEFAULT_DATETIME).toDate()) ||
-  moment.utc().toDate();
+const getCurrentDate = () => {
+  if (helpers.TEST_MODE) {
+    return new Date(new Date('2019-07-20').setUTCHours(0, 0, 0, 0));
+  }
+  if (helpers.DEV_MODE && process.env.REACT_APP_DEBUG_DEFAULT_DATETIME) {
+    return new Date(new Date(process.env.REACT_APP_DEBUG_DEFAULT_DATETIME).setUTCHours(0, 0, 0, 0));
+  }
+  return new Date();
+};
+
+/**
+ * Sets the UTC time to the end of day.
+ *
+ * @param {Date} date The date tp use
+ * @returns {Date} The date with the time set to the last millisecond of that day.
+ */
+const setEndOfDay = date => new Date(date.setUTCHours(23, 59, 59, 999));
+
+/**
+ * Sets UTC time to beginning of the day.
+ *
+ * @param {Date} date The date tp use
+ * @returns {Date} Returns the date with the time set to the start of that day.
+ */
+const setStartOfDay = date => new Date(date.setUTCHours(0, 0, 0, 0));
+/**
+ *Sets the UTC date and time to the end of tha month.
+ *
+ * @param {Date} date The date tp use
+ * @returns {Date} The date with the date and time set to the last millisecond of that month.
+ */
+const setEndOfMonth = date => new Date(setEndOfDay(date).setUTCMonth(date.getUTCMonth() + 1, 0));
 
 /**
  * Set a date range based on a granularity type.
  *
  * @param {object} params
  * @param {Date} params.date Start date, typically the current date.
  * @param {number} params.subtract Number of granularity type to subtract from the current date.
- * @param {string} params.measurement Granularity type.
- * @param {string} params.endOfMeasurement Granularity type.
+ * @param {'days' | 'weeks' | 'months' | 'years'} params.measurement Granularity type .
  * @returns {{endDate: Date, startDate: Date}}
  */
-const setRangedDateTime = ({ date, subtract, measurement, endOfMeasurement = 'days' }) => ({
-  startDate: moment.utc(date).startOf(measurement).subtract(subtract, measurement).toDate(),
-  endDate: moment.utc(date).startOf(measurement).endOf(endOfMeasurement).toDate()
+const setRangedDateTime = ({ date = getCurrentDate(), subtract = 0, measurement = 'days' }) => {
```

Comment:
this was something that was overlooked in the original and since we don't really run `setRangedDateTime` as `setRangedDateTime()` anywhere this doesn't really show up readily...  adding this fix here... 
```
... days' } = {})
```
lets the function run standalone with the parameter defaults...

(if you add a unit test for running `setRangedDateTime()` you'll see it)

Reply:
fixed.